src/main/java/duke/data/TaskList.java

```diff
@@ -0,0 +1,76 @@
+package duke.data;
+
+import duke.commands.Task;
+import duke.exceptions.DukeException;
+
+import java.util.ArrayList;
+
+public class TaskList {
+    private ArrayList<Task> lst;
+
+    public TaskList() {
+        lst = new ArrayList<Task>();
+    }
+
+    public void iterList() {
```

Comment:
Should this be renamed to something along the lines of printTaskList(), so that it more accurately describes what the function does? Because this function actually prints to the screen, instead of just iterating through the list.